package/MDAnalysis/coordinates/H5MD.py

```diff
@@ -0,0 +1,420 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+#
+# MDAnalysis --- https://www.mdanalysis.org
+# Copyright (c) 2006-2017 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# R. J. Gowers, M. Linke, J. Barnoud, T. J. E. Reddy, M. N. Melo, S. L. Seyler,
+# D. L. Dotson, J. Domanski, S. Buchoux, I. M. Kenney, and O. Beckstein.
+# MDAnalysis: A Python package for the rapid analysis of molecular dynamics
+# simulations. In S. Benthall and S. Rostrup editors, Proceedings of the 15th
+# Python in Science Conference, pages 102-109, Austin, TX, 2016. SciPy.
+# doi: 10.25080/majora-629e541a-00e
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+"""H5MD trajectories --- :mod:`MDAnalysis.coordinates.H5MD`
+========================================================
+
+The `H5MD`_ file format is based upon `HDF5`_, which makes use of parallel
+file system features through the MPI-IO interface of the HDF5 library.
+The reader currently uses the `H5PY`_ library to access data from an H5MD file.
```

Comment:
```suggestion
The `H5MD`_ trajectory file format is based upon the general, high performance `HDF5`_ file format.
HDF5 files are self documenting and can be accessed with the `h5py`_ library.
The HDF5 library (and `h5py`_) must be installed; otherwise, H5MD files cannot be read by MDAnalysis.

HDF5 can make use of parallel file system features through the MPI-IO interface of the HDF5 library
to improve parallel reads and writes.
```

By the way, what happens if h5py is not installed? You could also say here what happens in this case, e.g., what exception is raised.